plugin/src/integrationTest/kotlin/com/vanniktech/maven/publish/ProjectSpecRunner.kt

```diff
@@ -124,12 +125,7 @@ private fun ProjectSpec.publishingBlock(options: TestOptions): String {
   }
 }
 
-private fun ProjectSpec.writeSettingFile(path: Path, options: TestOptions) {
-  val rootProjectName = if (options.config == TestOptions.Config.DSL) {
-    artifactId
-  } else {
-    "default-root-project-name"
-  }
```

Comment:
This was a workaround to not beeing able to set the artifact id through the dsl which is now not needed anymore.